packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -444,6 +449,11 @@ class AndroidCameraCameraX extends CameraPlatform {
               'video recording while no recording is in progress.');
     }
     if (videoOutputPath == null) {
+      // Stop the current active recording as we will be unable to complete it
+      // in this error case.
+      recording!.close();
+      recording = null;
+      pendingRecording = null;
```

Comment:
Same here about adding it to the doc

Reply:
Added a note on the cleanup of the recording objects in the doc